### ISSUE TITLE
MuteVideo work in local, but not working in remote!!  :(

### ISSUE THREAD

**Issue by Garam-Leee** [AgoraIO-Community/Agora-RTC-React#36]
Left: web page made by me as Agora RTC NG SDK React Wrapper
Right: web page web demo serving at [https://webdemo.agora.io/basicVideoCall/index.html](https://webdemo.agora.io/basicVideoCall/index.html)

even if click muteVideo in left, video still show on right page
(but in left, disappear)
<img width="1625" alt="img" src="https://user-images.githubusercontent.com/101608868/195310931-fd4c952b-7a81-4137-8c69-6477193413b8.png">

Thank you for reading. 

I look forward to hearing from you soon. 🙌🏻

**Comment by EkaanshArora** [comment]
Are you using the example from the repo? Let me try to reproduce, thanks for flagging this.

**Comment by Garam-Leee** [comment]
Thank you for the quick comments 
I put my code on the repo below Please check it out. :) 
Thank you for your attention.

https://github.com/Garam-Leee/agora_test

**Comment by laviedegeorge** [comment]
@Garam-Leee Hello! Please how were you able to resolve this issue? I am currently having the same issue. cc @EkaanshArora

**Comment by bioDanCho** [comment]
@Garam-Leee @laviedegeorge Were you able to find a fix for this issue?

**Comment by laviedegeorge** [comment]
@bioDanCho yes I found a solution. The issue was from my code. What's your issue like?

**Comment by bioDanCho** [comment]
@laviedegeorge Locally, I can mute my mic and video, but the other users in the call can still see and hear me. It is the same issue as @Garam-Leee. How were you able to solve your issue?

**Comment by laviedegeorge** [comment]
@bioDanCho mine was a `useEffect` issue. Yours may be different. we can have a call now if you are free and we can look at it together. send a google meet link.

**Comment by bioDanCho** [comment]
@laviedegeorge Great! https://meet.google.com/rvt-pzso-hbm?pli=1

**Comment by Fray117** [comment]
care to send a snippets? @laviedegeorge

**Comment by laviedegeorge** [comment]
sure @Fray117 check [this](https://github.com/laviedegeorge/agora-react-example) out. It is a rough example. You can fork it and play around with it. If you need anything else, you can reach out.

**Comment by Fray117** [comment]
hey @laviedegeorge thanks for reaching me back, I've seen your code and noticing that you were using agora-rtc-sdk-ng as if creating a new wrapper, my colleague was using direct approach to rtc-sdk and were fine in live mode, i were still using the rtc mode inside rtc-react and this issue was found.

Question is, is this wrapper was not good enough or I was wrong?

in my case, I was using both setEnabled & also setMuted method (I'm not doing both at the same times as in documented said not to do so) and nothing as expected, not publishing any tracks at all even the enabled or muted status.

Any thought about this problem?

**Comment by laviedegeorge** [comment]
Hi @Fray117 i really can't say, but I used the react rapper and the this one but switched to this. I can't remember why I did so but it worked fine for me. If you are available, we can have a google meet call so I can see your code.

**Comment by Fray117** [comment]
guess I'm available since working on this too, you could join me on https://meet.google.com/rqe-spqw-sfz if that possible
